tests/at_tests.cpp

```diff
@@ -26,39 +26,39 @@ SUITE(at_tests)
 {
     TEST(static_array)
     {
-        int a[] = { 1, 2, 3, 4 };
+        size_t a[] = { 1, 2, 3, 4 };
 
-        for (int i = 0; i < 4; ++i)
+        for (size_t i = 0; i < 4; ++i)
             CHECK(at(a, i) == i+1);
```

Comment:
Good question. I agree that should be `4U` and not `4` but yeah, it's not mad about that. I didn't try with Clang so it might just be GCC being nice.

Do you want me to change that?